includes/gateways/paypal-standard.php

```diff
@@ -253,6 +256,9 @@ function give_process_paypal_web_accept( $data, $payment_id ) {
 		return;
 	}
 
+	// Register message in donation note that IPN received successfully.
+	give_insert_payment_note( $payment_id, __( 'Last IPN Received', 'give' ) );
```

Comment:
This is not particularly useful either. To be useful it should just "IPN received for {transaction ID}. Status {auth_status}" - See https://developer.paypal.com/docs/classic/ipn/integration-guide/IPNandPDTVariables/#transaction-and-notification-related-variables

Have you used Postman before to spoof the IPN? I can help you with this if not.